concent_api/core/views.py

```diff
@@ -562,22 +562,22 @@ def handle_send_force_payment(request, client_message: message.concents.ForcePay
             reason = message.concents.ForcePaymentRejected.REASON.NoUnsettledTasksFound
         )
     elif sum_of_payments > 0:
-        base.make_payment_to_provider(sum_of_payments, payment_ts, requestor_ethereum_public_key, client_public_key)
+        (amount_paid, amount_pending)   = base.make_payment_to_provider(sum_of_payments, payment_ts, requestor_ethereum_public_key, client_public_key)
```

Comment:
I know it was like this before but could you please split arguments into separate lines ? This is getting a little bit too long.

Reply:
I think that `make_payment_to_provider` should return only `amount_paid`, and `amount_pending` should be calculated here.

Reply:
Done

Reply:
@kbeker What I have meant is that `amount_pending` should be calculated `here`, in view. Sorry, maybe I should write this more precisely. I guess there will be no API method for getting something like `amount_pending`. In this view you have `sum_of_payments` and `amount_paid` so you can calculate the value. This is actually quite important to be done this way so please fix. In tests you can manipulate `base.payment_summary` and `base.make_payment_to_provider` to test different scenarios.